PluginFramework/Update/BaseUpdate.cs

```diff
@@ -29,14 +28,14 @@ protected BaseUpdate(PluginInformation information)
         /// <inheritdoc/>
         public void ChangeSettings(PluginSettings settings)
         {
-            this.Settings = settings;
+            Settings = settings;
         }
 
         /// <inheritdoc/>
-        public virtual UserControl GetSettingsPage()
-        {
-            return null;
-        }
+        //public virtual UserControl GetSettingsPage()
+        //{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5634024)